go/cmd/server/logging/filelogstorage.go

```diff
@@ -0,0 +1,74 @@
+package logging
+
+import (
+	"errors"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/apache/arrow/go/v8/arrow/array"
+	"github.com/apache/arrow/go/v8/parquet"
+	"github.com/apache/arrow/go/v8/parquet/pqarrow"
+)
+
+type FileLogStorage struct {
+	// Feast project name
+	project string
+	path    string
+}
+
+func NewFileOfflineStore(project string, offlineStoreConfig map[string]interface{}) (*FileLogStorage, error) {
```

Comment:
can we pass offlineStoreConfig as some struct with predefined properties rather than map with interface?

Reply:
Also please add a docstring here. We should at least state that this implementation is for testing

Reply:
Done.

Reply:
Done.